{{cookiecutter.bot_project_name}}/Dockerfile

```diff
@@ -32,11 +27,20 @@ COPY poetry.lock pyproject.toml ./
 RUN pip install --user --no-cache-dir poetry==1.1.6 && \
   poetry config virtualenvs.in-project true
 
+{% if cookiecutter.has_private_dependencies == "False"  %}
```

Comment:
`== "True"`? (and swap `if` and `else`) Also, can we use boolean?

Reply:
We can't use boolean